examples/ancillary_cultivated_2021.json

```diff
@@ -0,0 +1,74 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "ancillary_cultivated_2021",
+  "properties": {
+    "proj:epsg": 5070,
+    "proj:transform": [
+      30.031953583617746,
+      0.0,
+      -980.7863,
+      0.0,
+      -30.073848022599204,
+      731790.444
+    ],
+    "proj:shape": [
+      177,
+      293
+    ],
+    "datetime": "2022-10-12T11:08:03.126742Z"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          -95.919263,
+          29.613144
+        ],
+        [
+          -95.919216,
+          29.661152
+        ],
+        [
+          -96.010139,
+          29.661181
+        ],
+        [
+          -96.010133,
+          29.613174
+        ],
+        [
+          -95.919263,
+          29.613144
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "self",
+      "href": "/Users/pholleway/Code/Datasets/usda-cdl/examples/ancillary_cultivated_2021.json",
+      "type": "application/json"
+    }
```

Comment:
The examples shouldn't have self links, since they contain laptop-specific paths. See https://github.com/stactools-packages/noaa-cdr/blob/db0dbe718fe50b712dbcdb5aac56a081531da18a/scripts/create_examples.py#L109 for an example of saving a self-contained catalog, which removes all `self` links, or https://pystac.readthedocs.io/en/stable/api/pystac.html#pystac.STACObject.save_object for how to save an individual object w/o a self link.